app/controllers/api/v1/migrations/external_controller.rb

```diff
@@ -38,17 +38,17 @@ def create_role
         # Does: Creates a user.
 
         def create_user
-          language = if user_params[:language].blank? || user_params[:language] == 'default'
-                       I18n.default_locale
-                     else
-                       user_params[:language]
-                     end
+          user_hash = user_params.to_h
```

Comment:
This both simples and optimizes the logic,
Calling each time `user_params` is "somehow" an expensive operation compared to caching a hashed version of the params,
Besides we can use the hash for more efficient data retrieval and also assigning,
Using memoization here is an over kill since we may propagate it to all `*_params` methods,